File: src/main/kotlin/br/com/zup/beagleui/framework/ui/StyledWidget.kt
package br.com.zup.beagleui.framework.ui

import br.com.zup.beagleui.framework.core.NativeWidget
import br.com.zup.beagleui.framework.core.Widget

data class Border (
    val color: Long? = null,
    val radius: Double? = null,
    val size: Double? = null
)

data class StyledWidget (
    val border: Border? = null,
    val color: Long? = null,
    val child: Widget
) : NativeWidget()

File: Sources/BeagleUI/Sources/UIModels/StyledWidget.swift
//
//  StyledView.swift
//  BeagleUI
//
//  Created by Daniel Tes on 12/09/19.
//  Copyright © 2019 Daniel Tes. All rights reserved.
//

public struct StyledWidget: Widget {
    let border: Border?
    let color: String?
    let child: Widget?
}

public struct Border {
    let color: String?
    let radius: Double?
    let size: Double?
}
